### PR TITLE
[feat] Add AimLogger for CatBoost

### PR DESCRIPTION
**Pull request by devfox-se**

### CODE REVIEW

**Comment by devfox-se** [comment]
Implements #1680

File: aim/sdk/adapters/catboost.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    @property
+    def experiment(self) -> Run:
+        if not self._run:
+            self.setup()
+        return self._run
+
+    def setup(self):
+        if self._run:
+            return
+        if self._run_hash:
```
Comment (alberttorosyan) [inline aim/sdk/adapters/catboost.py:39]: do we need the re-opening logic here? as I understand, the callback is used as a logger (much like the  stdout) so normally, it should be just deleted once the process is finished.

File: aim/sdk/adapters/catboost.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
+from sys import stdout
+from typing import Optional
+
+from aim import Run
+from aim.ext.resource import DEFAULT_SYSTEM_TRACKING_INT
+
+
+class AimCallback:
```
Comment (alberttorosyan) [inline aim/sdk/adapters/catboost.py:8]: lets rename to `AimLogger`

**Review by alberttorosyan — CHANGES_REQUESTED** [review]
looks great! a couple of comments left

File: aim/sdk/adapters/catboost.py (hunk 1 above)
Comment (devfox-se) [reply aim/sdk/adapters/catboost.py:39]: leaving it wont have any negative impact I assume

**Review by alberttorosyan — APPROVED** [review]

**Pull request merged**